### PR TITLE
feat(n-card): adds `content-class` `footer-class` `header-class` .....

### PR DESCRIPTION
**Pull request by jahnli**

### CODE REVIEW

**Comment by jahnli** [comment]
对 `tailwindcss`  `unocss` 用户更友好

File: src/card/demos/enUS/index.demo-entry.md [hunk 1]
```diff
@@ -27,10 +27,14 @@ embedded.vue
 | --- | --- | --- | --- | --- |
 | bordered | `boolean` | `true` | Whether to show the card border. |  |
 | closable | `boolean` | `false` | Is it allowed to close. |  |
+| content-class | `string` | `undefined` | The class of the card content area. |  |
```
Comment (07akioni) [inline src/card/demos/enUS/index.demo-entry.md:30]:
```suggestion
| content-class | `string` | `undefined` | The class of the card content area. |  NEXT_VERSION |
```

File: src/card/demos/enUS/index.demo-entry.md [hunk 2]
```diff
@@ -27,10 +27,14 @@ embedded.vue
...
 | content-style | `Object \| string` | `undefined` | The style of the card content area. |  |
 | embedded | `boolean` | `false` | Use a darker background color to show the embedding effect (only for bright themes) |  |
+| footer-class | `string` | `undefined` | The class of the bottom area of the card. |  |
```
Comment (07akioni) [inline src/card/demos/enUS/index.demo-entry.md:33]:
```suggestion
| footer-class | `string` | `undefined` | The class of the bottom area of the card. | NEXT_VERSION |
```

File: src/card/demos/enUS/index.demo-entry.md [hunk 3]
```diff
@@ -27,10 +27,14 @@ embedded.vue
...
 | footer-style | `Object \| string` | `undefined` | The style of the bottom area of the card. |  |
+| header-class | `string` | `undefined` | The class of the card head area. |  |
```
Comment (07akioni) [inline src/card/demos/enUS/index.demo-entry.md:35]:
```suggestion
| header-class | `string` | `undefined` | The class of the card head area. | NEXT_VERSION |
```

File: src/card/demos/enUS/index.demo-entry.md [hunk 4]
```diff
@@ -27,10 +27,14 @@ embedded.vue
...
 | header-style | `Object \| string` | `undefined` | The style of the card head area. |  |
+| header-extra-class | `string` | `undefined` | The class of the card head extra area. | 2.25.0 |
```
Comment (07akioni) [inline src/card/demos/enUS/index.demo-entry.md:37]:
```suggestion
| header-extra-class | `string` | `undefined` | The class of the card head extra area. | NEXT_VERSION |
```

File: src/card/demos/zhCN/index.demo-entry.md [hunk 5]
```diff
@@ -29,10 +29,14 @@ embedded-debug.vue
...
 | bordered | `boolean` | `true` | 是否显示卡片边框 |  |
 | closable | `boolean` | `false` | 是否允许关闭 |  |
+| content-class | `string` | `undefined` | 卡片内容区域的类名 |  |
 | content-style | `Object \| string` | `undefined` | 卡片内容区域的样式 |  |
 | embedded | `boolean` | `false` | 使用更深的背景色展现嵌入效果，只对亮色主题生效 |  |
+| footer-class | `string` | `undefined` | 卡片底部区域的类名 |  |
 | footer-style | `Object \| string` | `undefined` | 卡片底部区域的样式 |  |
+| header-class | `string` | `undefined` | 卡片头部区域的类名 |  |
 | header-style | `Object \| string` | `undefined` | 卡片头部区域的样式 |  |
+| header-extra-class | `string` | `undefined` | 卡片头部额外内容的类名 | 2.25.0 |
```
Comment (07akioni) [inline src/card/demos/zhCN/index.demo-entry.md:39]:
```suggestion
| header-extra-class | `string` | `undefined` | 卡片头部额外内容的类名 | NEXT_VERSION |
```

File: src/card/demos/zhCN/index.demo-entry.md [hunk 6]
```diff
@@ -29,10 +29,14 @@ embedded-debug.vue
 | --- | --- | --- | --- | --- |
 | bordered | `boolean` | `true` | 是否显示卡片边框 |  |
 | closable | `boolean` | `false` | 是否允许关闭 |  |
+| content-class | `string` | `undefined` | 卡片内容区域的类名 |  |
 | content-style | `Object \| string` | `undefined` | 卡片内容区域的样式 |  |
 | embedded | `boolean` | `false` | 使用更深的背景色展现嵌入效果，只对亮色主题生效 |  |
+| footer-class | `string` | `undefined` | 卡片底部区域的类名 |  |
 | footer-style | `Object \| string` | `undefined` | 卡片底部区域的样式 |  |
+| header-class | `string` | `undefined` | 卡片头部区域的类名 |  |
```
Comment (07akioni) [inline src/card/demos/zhCN/index.demo-entry.md:37]:
```suggestion
| header-class | `string` | `undefined` | 卡片头部区域的类名 | NEXT_VERSION |
```

File: src/card/demos/zhCN/index.demo-entry.md [hunk 7]
```diff
@@ -29,10 +29,14 @@ embedded-debug.vue
 | --- | --- | --- | --- | --- |
 | bordered | `boolean` | `true` | 是否显示卡片边框 |  |
 | closable | `boolean` | `false` | 是否允许关闭 |  |
+| content-class | `string` | `undefined` | 卡片内容区域的类名 |  |
 | content-style | `Object \| string` | `undefined` | 卡片内容区域的样式 |  |
 | embedded | `boolean` | `false` | 使用更深的背景色展现嵌入效果，只对亮色主题生效 |  |
+| footer-class | `string` | `undefined` | 卡片底部区域的类名 |  |
```
Comment (07akioni) [inline src/card/demos/zhCN/index.demo-entry.md:35]:
```suggestion
| footer-class | `string` | `undefined` | 卡片底部区域的类名 | NEXT_VERSION |
```

File: src/card/demos/zhCN/index.demo-entry.md [hunk 8]
```diff
@@ -29,10 +29,14 @@ embedded-debug.vue
 | --- | --- | --- | --- | --- |
 | bordered | `boolean` | `true` | 是否显示卡片边框 |  |
 | closable | `boolean` | `false` | 是否允许关闭 |  |
+| content-class | `string` | `undefined` | 卡片内容区域的类名 |  |
```
Comment (07akioni) [inline src/card/demos/zhCN/index.demo-entry.md:32]:
```suggestion
| content-class | `string` | `undefined` | 卡片内容区域的类名 | NEXT_VERSION |
```

**Pull request merged**